torch/csrc/distributed/rpc/ProcessGroupAgent.cpp

```diff
@@ -73,14 +73,18 @@ ProcessGroupAgent::ProcessGroupAgent(
       "Resolved worker rank ", workerRankIter -> second,
       " does not match ProcessGroup rank ", pg_->getRank());
 
-  names_.resize(nameMap_.size());
-  for (auto& entry : nameMap_) {
-    names_[entry.second] = entry.first;
-  }
   sendThread_ = std::thread(&ProcessGroupAgent::sendLoop, this);
   listenerThread_ = std::thread(&ProcessGroupAgent::listenLoop, this);
 }
 
+uint64_t ProcessGroupAgent::getId(const std::string& workerName) {
```

Comment:
Here it's an uint64_t but I can see this being a slightly richer structure some time in the future. Could you add a type alias on the RPCAgent class for this? E.g. `using worker_id_t = uint64_t`.

If it won't be a richer structure then at least it serves as a bit of documentation.